tests/FsCodec.SystemTextJson.Tests/AutoUnionTests.fs

```diff
@@ -22,6 +22,10 @@ let [<Xunit.Fact>] ``Basic characteristics`` () =
     test <@ Not { body = "A"; opt = None; list = [] } = serdes.Deserialize "{\"case\":\"Not\",\"body\":\"A\",\"list\":[]}" @>
     test <@ Not { body = "A"; opt = None; list = ["A"] } = serdes.Deserialize "{\"case\":\"Not\",\"body\":\"A\",\"list\":[\"A\"]}" @>
 
+let [<Xunit.Fact>] ``It throws on null strings`` () =
+    raises <@ serdes.Deserialize<string> "null" @>
```

Comment:
needs to go in the low level characterisation tests e.g. somewhere in `SedesTests.fs` and/or `StjCharacterization`